systems/data-plan/api-gateway/pkg/rest/router.go

```diff
@@ -19,6 +20,8 @@ import (
 	"github.com/wI2L/fizz/openapi"
 )
 
+var REDIRECT_URI = "https://data-plan.dev.ukama.com/swagger/#/"
```

Comment:
This doesn't look good. what happnes when somebody else deploy this for their org ?

Reply:
Thanks for the feedback @vthakur7f , Addressed